panel/config.py

```diff
@@ -126,6 +126,9 @@ class _config(_base_config):
     template = param.ObjectSelector(default=None, doc="""
         The default template to render served applications into.""")
 
+    template_kwargs = param.Dict(default={}, instantiate=True, doc="""
```

Comment:
If we allow this to be changed then it should sync the changes with the template when updated for consistency. If we don't want to allow that we should set this to `constant`.